frontend/packages/modelServing/src/extensions.ts

```diff
@@ -3,18 +3,20 @@ import type { ProjectDetailsTab } from '@odh-dashboard/plugin-core/extension-poi
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { SupportedArea } from '@odh-dashboard/internal/concepts/areas/types';
```

Comment:
```suggestion
// eslint-disable-next-line import/no-extraneous-dependencies
import { SupportedArea } from '@odh-dashboard/internal/concepts/areas/types';
```

This type is broken now for some reason?